test/integration/resources/controls/database_spec.rb

```diff
@@ -5,7 +5,7 @@
   sql = mysql_session('root', 'gsql')
 
   describe sql.query('show databases') do
-    its('stdout') { should match(/databass/) }
-    its('stdout') { should_not match(/datatrout/) }
+    its('stderr') { should match(/databass/) }
```

Comment:
These all should be the following
```suggestion
    its('output') { should match(/databass/) }
```